src/test/java/juja/microservices/integration/UsersIntegrationTest.java

```diff
@@ -22,36 +30,43 @@
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
 @RunWith(SpringRunner.class)
-public class UsersIntegrationTest extends BaseIntegrationTest{
+@Import(DBUnitConfig.class)
+@TestExecutionListeners({DependencyInjectionTestExecutionListener.class,
+        DirtiesContextTestExecutionListener.class,
+        TransactionalTestExecutionListener.class,
+        DbUnitTestExecutionListener.class})
+@DbUnitConfiguration(databaseConnection = {"usersConnection", "crmConnection"})
+public class UsersIntegrationTest extends BaseIntegrationTest {
 
     private static final String USERS_URL = "/v1/users";
     private static final String USERS_BY_UUIDS_URL = "/v1/users/usersByUuids";
     private static final String USERS_BY_SLACK_NAMES_URL = "/v1/users/usersBySlackNames";
+    private static final String USERS_UPDATE_URL = "/v1/users/update";
     private static final String Fake_URL = "/fake";
```

Comment:
all constants should be in the same style - upper case

Reply:
Done!